gcp/modules/dcos-tested-gcp-oses/platform/cloud/gcp/centos_7.3/setup.sh

```diff
@@ -0,0 +1,34 @@
+sudo setenforce 0 && \
+sudo sed -i --follow-symlinks 's/^SELINUX=.*/SELINUX=disabled/g' /etc/sysconfig/selinux
+sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
+[dockerrepo]
+name=Docker Repository
+baseurl=https://yum.dockerproject.org/repo/main/centos/7
+enabled=1
+gpgcheck=1
+gpgkey=https://yum.dockerproject.org/gpg
+EOF
+# sudo yum -y update --exclude="docker-engine*"
+sudo mkdir -p /etc/systemd/system/docker.service.d
+sudo tee /etc/systemd/system/docker.service.d/override.conf <<- EOF
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon --storage-driver=overlay
+EOF
+sudo yum install -y docker-engine-1.13.1
+sudo systemctl start docker
+sudo systemctl enable docker
+sudo systemctl stop dnsmasq.service
+sudo systemctl disable dnsmasq.service
+sudo yum install -y wget
+sudo yum install -y git
+sudo yum install -y unzip
+sudo yum install -y curl
+sudo yum install -y xz
+sudo yum install -y ipset
+sudo yum install -y ntp
```

Comment:
while i am not a bash expert, would not it be easier to list all dependancies in one place, and then
install all of them with a command
`sudo yum install -t <command1 command2 command3>` ?

Reply:
This section was reused by this portion of DC/OS here: https://github.com/dcos/dcos/blob/master/dcos_installer/action_lib.py#L431-L436

You are correct, we can make this in one line. I assumed that readability was in a higher priority over completeness. 

Let me know which one you perfer in this case.